src/colorizer/loaders/UrlArrayLoader.ts

```diff
@@ -1,105 +1,80 @@
-import { parquetRead } from "hyparquet";
-import { compressors } from "hyparquet-compressors";
+// sort-imports-ignore
 import { DataTexture } from "three";
+import workerpool from "workerpool";
 
-import { FeatureArrayType, FeatureDataType, featureTypeSpecs } from "../types";
-import { nanToNull } from "../utils/data_utils";
+// Vite import directive for worker files! See https://v3.vitejs.dev/guide/features.html#import-with-query-suffixes.
```

Comment:
Note that Vite's documentation claims that `import` in workers will be fine in production but fail everywhere but Chrome in dev mode? But I've tested this locally in Firefox and Edge without seeing any issues.

Reply:
Wait nevermind that was vite v3